modules/claims_api/lib/library_base.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'claims_api/claim_logger'
+require 'custom_error'
+
+module ClaimsApi
+  class LibraryBase
+    def error_handler(error, async = true) # rubocop:disable Style/OptionalBooleanParameter
+      ClaimsApi::CustomError.new(error, async).build_error
```

Comment:
In what case might `async` be `false`? As written, we only call `error_handler` on line 25 with one param, meaning `async` will always be `true`. Would it make sense to write something like:

```suggestion
    def error_handler(error)
      ClaimsApi::CustomError.new(error, true).build_error
```

or even more succinctly, replace line 25 with `ClaimsApi::CustomError.new(e, true).build_error` and remove this method?